lib/src/main/java/space/taran/arklib/data/preview/generator/VideoPreviewGenerator.kt

```diff
@@ -61,10 +67,13 @@ object VideoPreviewGenerator : PreviewGenerator {
                         }
                     }
                     bitmap
-                } ?: Glide.with(app.baseContext).asBitmap()
-                    .load(path.toFile())
-                    .diskCacheStrategy(DiskCacheStrategy.NONE)
-                    .submit().get()
+                } ?: let {
+                    val request = ImageRequest.Builder(app)
+                        .data(path.toFile())
+                        .build()
+
+                    ImageUtils.arkImageLoader.execute(request).drawable!!.toBitmap()
```

Comment:
Please handle the case when `drawable` is `null` here, otherwise there will be crash in case the load result returns a `null` drawable.

Reply:
It's already wrapped in `Result`